docs/UserGuide.md

```diff
@@ -155,10 +155,13 @@ For the good majority of visual learners, we figured to present information in a
 
 3. Copy the file to the folder you want to use as the home folder for your ATAS. All the data related to ATAS will be stored here.
 
-4. Double-click the file to start the app. The GUI similar to the below image should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![StartPage](images/StartPage.png)
+4. Double-click the file to start the app. The GUI similar to the below image should appear in a few seconds.<br>
+   ![WelcomePage](images/user-guide/3.1-WelcomePage.png)
 
-5. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
+5. Now click on the Student tab and notice how the app contains some sample data. <br/>
+   ![StudentSampleData](images/user-guide/3.2-StudentSampleData.png)
```

Comment:
Maybe figure descriptions could be added?

Reply:
I've added them in my PR!